SPECS/mstflint/mstflint.spec

```diff
@@ -57,7 +93,17 @@ find %{buildroot} -type f -name '*.a' -delete
 %{_datadir}/mstflint
 %{_mandir}/man1/*
 
+%post kernel
+depmod %{kver} -a
+
+%files kernel
+%defattr(-,root,root,-)
+/lib/modules/%{kver}/%{install_mod_dir}/
```

Comment:
It looks like we're repeating `/lib/modules/%{kver}/%{install_mod_dir}/` multiple times in the spec. Let's move it to a macro to avoid repetitions and accidental issues related to typos or forgetting to change one of the repeated paths, if ever needed in the future. I think the `install_mod_dir` macro is used only in this path, so we might replace it as well.